charts/prometheus-thanos/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.4.0"
 description: A Helm chart for thanos monitoring components
 name: prometheus-thanos
-version: 0.8.0
+version: 0.9.0
```

Comment:
Has to be a major version increase. Because a simple `helm upgrade` won't work...